typings/telegraf.d.ts

```diff
@@ -0,0 +1,213 @@
+/// <reference types="node" />
+
+import { IncomingMessage, ServerResponse } from 'http'
+import * as https from 'https'
+import * as http from 'http'
+import { TlsOptions } from 'tls'
+
+import * as tt from './telegram-types.d'
+
+import { ContextMessageUpdate } from './context'
+import { Composer } from './composer'
+import { Telegram, TelegramOptions } from './telegram'
+
+interface TelegrafOptions {
+  /**
+   * Telegram options
+   */
+  telegram?: TelegramOptions
+
+  /**
+   * Bot username
+   */
+  username?: string
+}
+
+interface LaunchPollingOptions {
+  /**
+   * Poll timeout in seconds
+   */
+  timeout?: number
+
+  /**
+   * Limits the number of updates to be retrieved
+   */
+  limit?: number
+
+  /**
+   * List the types of updates you want your bot to receive
+   */
+  allowedUpdates?: tt.UpdateType[] | tt.UpdateType | null
+
+  /**
+   * Polling stop callback
+   */
+  stopCallback?: () => void | null
+}
+
+interface LaunchWebhookOptions {
```

Comment:
I think these interfaces should also be exported (in this file, not necessarily from index)

Reply:
I don't think it's necessary for a function you only call once. But whatever, I'll do it anyway.

Reply:
I don't care about the specific interfaces. They have some content. If I have to initialize larger objects I start to split them up. While doing so I want to have type checking as early as possible.
I don't know if that might happen with exactly these but they are publicly facing interfaces meant to be used by the user so they should be public.

Reply:
If a user is able to use the interface it should be exported.